examples/timeline/groups/nestedGroups.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <title>Timeline | Nested Groups example</title>
+
+  <style>
+    body, html {
+      font-family: arial, sans-serif;
+      font-size: 11pt;
+    }
+
+    #visualization {
+      box-sizing: border-box;
+      width: 100%;
+      height: 300px;
+    }
+  </style>
+
+  <!-- note: moment.js must be loaded before vis.js, else vis.js uses its embedded version of moment.js -->
+  <script src="http://cdnjs.cloudflare.com/ajax/libs/moment.js/2.8.4/moment.min.js"></script>
+
+  <script src="../../../dist/vis.js"></script>
+  <link href="../../../dist/vis-timeline-graph2d.min.css" rel="stylesheet" type="text/css" />
+  <script src="../../googleAnalytics.js"></script>
+</head>
+<body>
+<p>
+  This example demonstrate using groups. Note that a DataSet is used for both
+  items and groups, allowing to dynamically add, update or remove both items
+  and groups via the DataSet.
+</p>
+<div id="visualization"></div>
+
+<script>
+  var now = moment().minutes(0).seconds(0).milliseconds(0);
+  var itemCount = 60;
+
+  // create a data set with groups
+  var groups = new vis.DataSet();
+
+  groups.add([
+    {
+      id: 1,
+      content: "Lee",
+      nestedGroups: [11,12,13]
+    },
+    {
+      id: 2,
+      content: "Alson",
+    },
+    {
+      id: 3,
+      content: "John",
+      nestedGroups: [14],
+      showNested: false
```

Comment:
The `showNested: false` causes the bad error `Cannot redraw item: no parent attached`. Looks like Items can not be attached to groups that are not visible.
This needs to be fixed !